Mods/ArchipelagoPlayer.SC2Mod/enUS.SC2Data/LocalizedData/GameHotkeys.txt

```diff
@@ -207,6 +207,9 @@ Button/Hotkey/AP_MissilePods=E
 Button/Hotkey/AP_MissileTurret=T
 Button/Hotkey/AP_MorphBackToRoboticsFacility=M
 Button/Hotkey/AP_MorphBackToStargate=M
+Button/Hotkey/AP_Morphling=G
+Button/Hotkey/AP_MorphlingToBaneling=Z
+Button/Hotkey/AP_MorphlingToImpaler=F

```

Comment:
What about:
Ravager - R
Viper - V
Baneling - A
Impaler - E

Reply:
I'd prefer to change as few hotkeys as possible for morphling special-casing. Baneling Z/Impaler F means only two buttons change

Reply:
The thing is that these are more close to the unit names

Reply:
BTW E - Impaler is default IIRC

Reply:
E - Viper and E - Baneling are both also default, which is why I changed Baneling and Impaler for Morphling only. V - Ravager is default too, so your suggestion would change 3 hotkeys vs 2

Reply:
Yeah, but it feels more natural but I got another idea:
Impaler - R
Baneling - A